### ISSUE TITLE
Add an identifier to account cache data

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#68]
Merge into API response the following object: 

```js
{meta: {version: {major: x, minor: y, patch: p}}
```

Then in a future release (after 0.8.3) we will begin checking for it when validating cached account data. This is a long term solution to new users switching to Jakcodex/Muledump from alternate versions and getting a variety of bugs due to malformed cached account data.

**Comment by jakcodex** [comment]
Added to the end of the current data structure:

```js
meta: {
    token: setuptools.config.actoken,
    version: $.extend(true, {}, setuptools.version)
}
```

Token identifier to try and prevent any potential future conflicts.

**Comment by jakcodex** [comment]
Released in 0.8.3-0.
